opencti-platform/opencti-front/src/utils/Graph.js

```diff
@@ -394,6 +394,7 @@ export const defaultValue = (n, fallback = 'Unknown') => {
       || n.attribute_abstract
       || n.opinion
       || n.value
+      || n.product
```

Comment:
it's result_name that we want to display for malware analyses

Reply:
My bad, missed that. I assumed that the spread on the query was correct 😅
I'll fix that